src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -234,6 +234,11 @@
   <ItemGroup Condition="'$(TargetGroup)' != 'netcoreapp'">
     <Compile Include="System\StringSplitExtensions.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netfx'">
+    <Compile Include="$(CommonTestPath)\Tests\System\SpanExtensions.netstandard.cs">
```

Comment:
Why is this with a "netstandard" extension but for netfx?

Reply:
This helper file supports portable Span which is available in netstandard. In this case it's only used for netfx but if there would be another tfm that doesn't have System.Memory inbox but uses the System.Memory package it could use that helper file also. We can rename it to netfx in this case as we don't have another tfm that needs that.